lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -30,19 +30,17 @@ class MailboxController extends BaseController {
 
   final folderMailboxTree = MailboxTree(MailboxNode.root()).obs;
 
```

Comment:
Why not assign `final mailboxDashBoardController = Get.find<MailboxDashBoardController>();`
You always duplicate call `Get.find<MailboxDashBoardController>()`.